inc/core/patterns/cleanbc/header-cleanbc.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * CleanBC Header block pattern.
+ *
+ * @since 1.0.0
+ *
+ * @package Bcgov/Theme/Block
+ */
+return array(
+	'title'      => __( 'CleanBC Header', 'bcgov_blocks_theme' ),
+	'categories' => array( 'bcgov-blocks-theme-patterns-cleanbc' ),
+	'content'    => '<!-- wp:group {"style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"spacing":{"padding":{"top":"0px","right":"65px","bottom":"0px","left":"65px"}}},"borderColor":"primary-brand-yellow","backgroundColor":"primary-brand-blue","textColor":"white","className":"bcgov-header-group","layout":{"inherit":true}} -->
```

Comment:
I would have been more inclined to keep this in a separate  file, we could talk about this strategy, of it it makes more sense to keep it here.